### PR TITLE
add feature detection example, clean up code examples

### PR DESCRIPTION
**Pull request by patrickhlauke**
Closes https://github.com/w3c/pointerevents/issues/94

### CODE REVIEW

**Comment by patrickhlauke** [comment]
also adds a "don't forget about keyboard users" bit to the canvas drawing example (doesn't go into details, but as a gentle reminder of the note about not forgetting kbd users in the spec's intro)

File: index.html [hunk 1]
```diff
@@ -132,79 +132,106 @@
...
+    ...
+    target.addEventListener('mousedown', ...);
+    ...
+}
+
+// additional event listeners for keyboard handling
+...</code>
+</pre>
+<pre id="example_2" class="example" title="Detecting the type of input from a user">
+<code>window.addEventListener("pointerdown", detectInputType, false);
```
Comment (NavidZ) [inline index.html]: I can see this is the same as before. But do we need this "false" as the third parameter here at all? Just to make this line simpler I guess maybe we can get rid of it and not giving the false impression of this having anything to do with pointer types? WDYT?
This also appeared in a few other examples.

**Comment by NavidZ** [comment]
Looks good to me overall.

File: index.html (hunk 1 above)
Comment (patrickhlauke) [inline index.html]: agree, this somehow snuck into the original and stayed around. unnecessary in modern browsers...i'll nuke it (and any other instances i find). good catch

**Comment by patrickhlauke** [comment]
merging this later today, unless @RByers @scottgonzalez @teddink have any comments?

**Comment by teddink** [comment]
LGTM - thanks Patrick!

**Comment by RByers** [comment]
Thanks for this - retroactive post-vacation LGTM

**Pull request merged**